### DISCUSSION TITLE
[Weekly Review Report]: Oct 28 - 31

### DISCUSSION THREAD

**Discussion by AlexanderPico** [wikipathways/wikipathways-help#140] [curation-reports]
### Recent changes

 * 18 pathways - mostly edits from Eric _- thanks!_ 

### Under construction

Pathways ready to go live:
 * [WP5432](https://classic.wikipathways.org/index.php/Pathway:WP5432) - Added description and approved
 * [WP5469](https://classic.wikipathways.org/index.php/Pathway:WP5469) - Added description and approved
 * [WP5465](https://classic.wikipathways.org/index.php/Pathway:WP5465) - Added description and approved
 * [WP5477](https://classic.wikipathways.org/index.php/Pathway:WP5477) - Added description and approved
 * [WP5488](https://classic.wikipathways.org/index.php/Pathway:WP5488) - Added description, fixed title and approved

Pathways without much hope and not touched over over 3 months:
 * [WP5454](https://classic.wikipathways.org/index.php/Pathway:WP5454) - Marked for deletion
 * [WP5457](https://classic.wikipathways.org/index.php/Pathway:WP5457) - Marked for deletion
 * [WP5459](https://classic.wikipathways.org/index.php/Pathway:WP5459) - Marked for deletion
 * [WP5460](https://classic.wikipathways.org/index.php/Pathway:WP5460) - Marked for deletion
 * [WP5462](https://classic.wikipathways.org/index.php/Pathway:WP5462) - Marked for deletion
 * [WP5468](https://classic.wikipathways.org/index.php/Pathway:WP5468) - Marked for deletion

@khanspers This one looks ready to go live too, no? https://classic.wikipathways.org/index.php/Pathway:WP5490

### New pathways

Nice new pathway from Eric! 
* [AMPK regulation of mammary milk protein synthesis](https://classic.wikipathways.org/index.php/Pathway:WP5492) 

### Deleted Pathways

None

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by AlexanderPico** [comment]
Oh, and I removed the broken image retrieval from the QA protocol. So, it just grabs title, species and links now.

**Comment by khanspers** [comment]
Thanks for the great report! A few comments:

For [WP5490](https://classic.wikipathways.org/index.php/Pathway:WP5490), I still need to add some gene annotations for PI3K, PKC, AKT/mTOR. Will complete it in the next few days, thanks for the reminder.

[WP5432](https://classic.wikipathways.org/index.php/Pathway:WP5432): We were discussing this pathway (and [WP5424](https://classic.wikipathways.org/index.php/Pathway:WP5424)) with Marian Breuer (PI who's students worked on these pathways) in today's curators meeting. One of how students is writing up longer descriptive descriptions for both pathways. 

The pathways you marked for deletion are all from Marvin's student I believe. @marvinm2 please comment if any of these pathways should not be deleted.

**Reply by marvinm2** [reply]
Thanks for pinging me! 

These can indeed be removed (some are duplicates / mistakes of double upload)
[WP5457](https://classic.wikipathways.org/index.php/Pathway:WP5457) - Marked for deletion
[WP5459](https://classic.wikipathways.org/index.php/Pathway:WP5459) - Marked for deletion
[WP5460](https://classic.wikipathways.org/index.php/Pathway:WP5460) - Marked for deletion
[WP5462](https://classic.wikipathways.org/index.php/Pathway:WP5462) - Marked for deletion

The other ones:
[WP5454](https://classic.wikipathways.org/index.php/Pathway:WP5454) - updated and added to AOP collection
[WP5468](https://classic.wikipathways.org/index.php/Pathway:WP5468) - this is a useful list of genes. My current intern is looking into specifying inflammatory mediators per organ system, and will probably use this. I removed the deletion tag

**Reply by khanspers** [reply]
Great! Thank you Marvin. The first 4 have been deleted, and I added the Under Construction tag to WP5468 with a message that its being used for curation.
